src/main/webapp/registration.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
```

Comment:
I might be wrong but I think these are spaced with only a single space. Google HTML/CSS guidelines asks for 2. Not a big deal, just pointing it out.

Reply:
Thanks for pointing that out!